internal/librarian/generate.go

```diff
@@ -206,9 +208,24 @@ func (r *generateRunner) runGenerateCommand(ctx context.Context, outputDir strin
 			RepoDir:   r.repo.Dir,
 		}
 		slog.Info("Performing refined generation for library", "id", libraryID)
-		return libraryID, r.containerClient.Generate(ctx, generateRequest)
+		generateResponse := r.containerClient.Generate(ctx, generateRequest)
```

Comment:
I believe `generateResponse` is just the error? we would want to early return if so.